### PR TITLE
inline.d: use foreach, eliminiate -> and ::, trivial

### PR DESCRIPTION
**Pull request by WalterBright**
1. rewrite for loops to foreach
2. replace C++ -> and :: in comments with .

### CODE REVIEW

File: src/inline.d [hunk 1]
```diff
@@ -96,7 +96,7 @@ public:
     override void visit(CompoundStatement s)
     {
         scope InlineCostVisitor icv = new InlineCostVisitor(this);
-        for (size_t i = 0; i < s.statements.dim; i++)
+        foreach (i; 0 .. s.statements.dim)
```
Comment (yebblies) [inline src/inline.d]: Use `foreach (i, s2; *s.statements)`?

File: src/inline.d [hunk 2]
```diff
@@ -82,9 +82,9 @@ public:
 
     override void visit(Statement s)
     {
-        //printf("Statement::inlineCost = %d\n", COST_MAX);
```
Comment (yebblies) [inline src/inline.d]: It's actually been kind of nice being able to grep for `Class::`.  Though I guess `visit(Class` will work now.

File: src/inline.d (hunk 1 above)
Comment (WalterBright) [inline src/inline.d]: I kept this to the trivial, the number of diff lines was high enough.

File: src/inline.d (hunk 2 above)
Comment (WalterBright) [inline src/inline.d]: Time to leave C++ behind :-)

**Comment by yebblies** [comment]
Auto-merge toggled on

**Comment by WalterBright** [comment]
rebased

**Pull request merged**